src/viur/core/skeleton.py

```diff
@@ -1338,7 +1338,14 @@ def fromSkel(cls, kindName: str, *args: List[str]) -> Type[RefSkel]:
         """
         newClass = type("RefSkelFor" + kindName, (RefSkel,), {})
         fromSkel = skeletonByKind(kindName)
-        newClass.__boneMap__ = {k: v for k, v in fromSkel.__boneMap__.items() if k in args}
+        bone_map = {}
+        for arg in args:
+            if "*" in arg:  # we have a wild card
+                bone_map |= {k: v for k, v in fromSkel.__boneMap__.items() if k.startswith(arg.replace("*", ""))}
+            else:
+                if bone := fromSkel.__boneMap__.get(arg):
+                    bone_map[arg] = bone
+        newClass.__boneMap__ = bone_map
```

Comment:
That looks complicated. Couldn't we simplify this with [`utils.string.is_prefix()`](https://github.com/viur-framework/viur-core/blob/c34d45fdf0eeeace4d4317ab89cd251ddf6191cf/src/viur/core/utils/string.py#L88): 

```suggestion
        for arg in args:
            bone_map |= {k: v for k, v in fromSkel.__boneMap__.items()
                         if utils.string.is_prefix(k, arg, "*"))}
        newClass.__boneMap__ = bone_map
```

Reply:
But now we iter over the bonemap `len(args) `time ? That is not so good i think.

Reply:
> But now we iter over the bonemap `len(args) `time ? That is not so good i think.

well, okay.


But I think index access would be faster than an _in_ test. And slicing the prefix once would be better than _replace_ on every _if_ check. But I didn't bench marked this ...


```suggestion
        for arg in args:
            if arg[-1] == "*":  # we have a wild card prefix
                prefix = arg[:-1]
                bone_map |= {k: v for k, v in fromSkel.__boneMap__.items() if k.startswith(prefix)}
            else:
                if bone := fromSkel.__boneMap__.get(arg):
                    bone_map[arg] = bone
        newClass.__boneMap__ = bone_map
```